File: backend/components/metiers/threed/security/docs/samples/compliance/samples/sample_compliance.py
# sample_compliance.py
# Exemple d’intégration avancée pour la conformité

from helpers_compliance import check_compliance, generate_compliance_report

data = {'entity': 'ACME Corp', 'standards': ['ISO27001', 'SOC2']}
print('ISO27001:', check_compliance('ISO27001', data))
print(generate_compliance_report(data))

File: backend/components/metiers/threed/security/docs/samples/compliance/samples/sample_compliance.js
// sample_compliance.js
// Exemple d’intégration avancée pour la conformité

const { checkCompliance, generateComplianceReport } = require('./helpers_compliance');

const data = { entity: 'ACME Corp', standards: ['ISO27001', 'SOC2'] };
console.log('ISO27001:', checkCompliance('ISO27001', data));
console.log(generateComplianceReport(data));
